test-utils/build.gradle

```diff
@@ -18,12 +18,8 @@ dependencies {
         exclude group: 'org.apache.hive', module: 'hive-service'
     }
 
-    api group: 'junit', name: 'junit', version: '4.13.2'
-    api group: 'org.hamcrest', name: 'hamcrest-core', {
-        version {
-            strictly '2.2'
-        }
-    }
+    api group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
+    api group: 'org.hamcrest', name: 'hamcrest-library', version: '2.2'
```

Comment:
Verified this works, thanks @ncordon 🎉

Reply:
The reason is described here in case you are curious: http://hamcrest.org/JavaHamcrest/distributables#upgrading-from-hamcrest-1x